src/video/SDL_pixels.c

```diff
@@ -661,6 +661,7 @@ const SDL_PixelFormatDetails *SDL_GetPixelFormatDetails(SDL_PixelFormat format)
     }
 
     if (SDL_FindInHashTable(SDL_format_details, (const void *)(uintptr_t)format, (const void **)&details)) {
+        SDL_UnlockMutex(SDL_format_details_lock);
         return details;
```

Comment:
Would a `goto done` make sense here?
It avoids some code duplication.

Reply:
Oh yeah, that's even better. Want to commit that change?

Reply:
I pushed it in c80665a69

Reply:
Thanks!